backend/controllers/followups.ts

```diff
@@ -138,6 +139,26 @@ export function postSurvey(req: ajRequest, res: Response) {
   pollResult.postPollResult(req.followup, req.body)
 }
 
+export async function benefitActionSurveyTrackerUpdate(followup: any) {
+  const surveyTracker = followup.surveys.find(
+    (survey) => survey.type === SurveyType.trackClicBenefitActionEmail
+  )
+  if (!surveyTracker) {
+    const tracker = await followup.createSurvey(
+      SurveyType.trackClicBenefitActionEmail
+    )
+    followup.surveys.push(tracker)
+  }
+  await followup.updateSurvey(SurveyType.trackClicBenefitActionEmail, [])
+  await followup.save()
+}
+
+export async function accessSurvey(req: ajRequest, res: Response) {
+  const { followup } = req
+  await benefitActionSurveyTrackerUpdate(followup)
+  res.redirect(`${config.baseURL}${followup.surveyPath}`)
```

Comment:
```suggestion
  res.redirect(followup.surveyPath)
```

Est ce que ça fonctionne ? Ça serait plus simple.

Reply:
Oui ça fonctionne aussi